api/app/based58.py

```
@@ -0,0 +1,50 @@
+    "KMD": KMD_CoinParams,
+    "DOC": KMD_CoinParams,
+    "BTC": BTC_CoinParams,
+    "LTC": LTC_CoinParams
+}
+
+
+def calc_addr_from_pubkey(coin, pubkey):
+    bitcoin.params = COIN_PARAMS[coin]
+    try:
+        return str(P2PKHBitcoinAddress.from_pubkey(x(pubkey)))
+    except Exception as e:
```

Ruff (0.14.5)

Do not catch blind exception: `Exception`

(BLE001)